app/src/main/java/tttTDD/Java/ConsoleClient.java

```diff
@@ -0,0 +1,129 @@
+package tttTDD.Java;
+
+import java.util.Scanner;
+
+public class ConsoleClient {
```

Comment:
We talked about the refacoring book I think at some point. 

This might seem like a big class at the moment, but I think this would be a great example to use to practice some concepts in a chapter or two of the refactoring book I think we talked about at some point. 

I can see a few patterns of functionality in this class that could maybe be refactored (for another day / story)